packages/utilities/psammead-locales/moment/fa.test.js

```diff
@@ -0,0 +1,193 @@
+/* eslint-disable */
```

Comment:
Are there any tests in here that show the output for gregorian and jalali datestamps? I cant see them, but I dont speak farsi :P 🙈

Reply:
I applied some fixes, in the tests `test('format', function(){})` should show the calendar conversion for `ps` and `fa` . I don't speak farsi too 🙈

Reply:
Sorry i still cant see them. Surely we should have tests thats like `moment(time).format('LL')` assert equals `<gregorian> <jalialli>`?

Reply:
Also as far as I can see `test('format', function(){})` doesnt exist

Reply:
The test cases for dual calendar timestamps are here: https://github.com/bbc/psammead/pull/1844/files#diff-912d92b07d9011a084afbdaa2406e7c3R43-R45